Bash_scripts/HW10/script_for_find_files.sh

```diff
@@ -0,0 +1,9 @@
+
+#!/usr/bin/bash
+path_to_dir="$1"
+
+ls -laR "${path_to_dir}"/*.$2 \
```

Comment:
This will search only throw `path_to_dir` but not inside subdirectories.

Reply:
I remade script for find files inside subdirectories